index.bs

```diff
@@ -9024,6 +9070,386 @@ worker=] algorithm:
 
 </div>
 
+## The storage Module ## {#module-storage}
```

Comment:
Would you mind to move this whole section down so that we can keep an alphabetical order of modules? Thanks!

Reply:
@whimboo I'm happy to move the section, but since the list is not sorted alphabetically, I need more specific instruction about where it the storage module should go.

```
$$('[id^=module-]').map(el => el.innerText).join('\n')
"module name
6.1. The session Module
6.1.1. Definition
6.1.2. Types
6.1.3. Commands
6.2. The browser Module
6.2.1. Definition
6.2.2. Commands
6.3. The browsingContext Module
6.3.1. Definition
6.3.2. Types
6.3.3. Commands
6.3.4. Events
6.4. The network Module
6.4.1. Definition
6.4.3. Types
6.4.4. Commands
6.4.5. Events
6.5. The script Module
6.5.1. Definition
6.5.3. Types
6.5.4. Commands
6.5.5. Events
6.6. The storage Module
6.6.1. Definition
6.6.2. Types
6.6.3. Commands
6.7. The log Module
6.7.1. Definition
6.7.2. Types
6.7.3. Events
6.8. The input Module
6.8.1. Definition
6.8.2. Types
6.8.3. Commands" 
```

Reply:
Oh, I thought that only `session` is not sorted. But given that also `log` and `input` are affected I would suggest we keep it after `script` and once this PR is merged we can re-organize the list of modules.

Reply:
Maybe we could rename session to "000_session" while we're at it.